### DISCUSSION TITLE
[Taskbar Labels for Windows 11] Auto set item width by auto detect label length.

### DISCUSSION THREAD

**Discussion by HumanMus1c** [ramensoftware/windhawk-mods#627] [Mod ideas]
It's probably better?

**Comment by m417z** [comment]
Are you talking about [Taskbar Labels for Windows 11](https://windhawk.net/mods/taskbar-labels)? That's the Windows default, and there's an option for that in the mod, unless I misunderstood what you're asking.

**Reply by HumanMus1c** [reply]
yes.
